signedjson/__init__.py

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from importlib_metadata import version, PackageNotFoundError
+try:
+    from importlib.metadata import version, PackageNotFoundError
+except ImportError: # pragma: nocover
```

Comment:
```suggestion
except ImportError:  # pragma: nocover
```

We require two spaces before a comment that's on the same line as code. Let's see if this works!

Reply:
thanks!